compiler/ras/Debug.cpp

```diff
@@ -4012,6 +4012,7 @@ TR_Debug::getRuntimeHelperName(int32_t index)
          case TR_S390interfaceCallHelper:                          return "_interfaceCallHelper";
          case TR_S390interfaceCallHelperSingleDynamicSlot:         return "_interfaceCallHelperSingleDynamicSlot";
          case TR_S390interfaceCallHelperMultiSlots:                return "_interfaceCallHelperMultiSlots";
+         case TR_S390interfaceCallHelperReadOnlyMultiSlots:        return "_interfaceCallHelperReadOnlyMultiSlots";
```

Comment:
We should stick with the "ReadOnly" naming convention as the suffix for easier searchability. See `TR_S390virtualUnresolvedHelperReadOnly` as an example.

Reply:
Yeah that would be more consistent, moved `ReadOnly` at the end of helper name for consistency in https://github.com/eclipse/openj9-omr/compare/64d79bb370b56efb357c9e216b1cdb37d46b02dc..0c164e76447b1993a9d768bd58b12153f37aa0e8

Reply:
Perhaps a larger question to raise in a different forum (like a separate issue), but I've found using "RWX" and "RX" (for writable code cache and readonly code cache, respectively) convenient shorthands for helper names and file names.  The x86 code in the openj9-omr and openj9 repos doesn't reflect that, but in the work I've been doing to contribute to the master branches I've started using them.  Not sure if you have any opinions on that naming scheme.